tests/unit-tests/admin/test-class-sensei-setup-wizard.php

```diff
@@ -214,8 +214,9 @@ public function testActivationRedirect() {
 		// Create and login as administrator.
 		$admin_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
 		wp_set_current_user( $admin_id );
+		set_current_screen( 'dashboard' );
 
-		set_transient( 'sensei_activation_redirect', 1, 30 );
+		update_option( 'sensei_activation_redirect', 1 );
```

Comment:
Instead of using mock here, is it possible to test it using the `Sensei_Test_Redirect_Helpers` trait? You can see an example here https://github.com/Automattic/sensei/blob/0132d5c48c67abb440bab859fd83d7c4654d75df/tests/unit-tests/admin/home/tasks/task/test-class-sensei-home-task-pro-upsell.php#L54-L69 . Maybe it can farther improve the reliability of the test? Just a suggestion, not necessary at all

Reply:
Sure! Thank you for the suggestion! Updated here: https://github.com/Automattic/sensei/pull/7581/commits/693851ea06dfbb787ccb31db0826f6c19cf648ae

Reply:
Very nice! <3 Now they are covering much more realistically than the mock!